focuspoints.lrdevplugin/DefaultDelegates.lua

```diff
@@ -102,7 +102,7 @@ end
 
 --[[
   -- method figures out the orientation the photo was shot at by looking at the metadata
-  -- returns 90, 270, or 0 (in trigonometric sense)
+  -- returns the rotation in rad in trigonometric sense
```

Comment:
Yes, I thought about your very last argument yesterday as I was programming also.
Let me change it everywhere accordingly to degrees.